File: CODE/atcoder.java
class atcoder{
    public static void main(String args[]){
     Scanner sc=new Scanner(System.in);
      String str=sc.nextLine();
      char a;
      for(int i=0;i<str.length();i++){
          a=str.charAt(0);
          if (str.charAt(i)==a){
              System.out.println("Lost");
              break;
          }
          else
            continue;
      }
      System.out.println("Won");
    }
    }

File: CODE/misc/atcoder.cpp
#include<iostream>
using namespace std;
int main(){
    string str;
    cin>>str;
    for(int i=0;i<str.length();i++){
        if((str[i]==str[i+1]) && (str[i+1]==str[i+2])){
            cout<<"Won";
            break;
        }
        else{
            cout<<"Lost";
            break;
        }
    }
}